app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Public Recipes</h1>
+
+<div id="recipes">
+<% if @recipes.present? %>
+  <% @recipes.each do |recipe| %>
+    <div class="d-flex flex-row  border border-primary mb-3 p-2">
+    <div class="d-flex flex-column flex-shrink-0 me-5 ms-2" >
+    <%= link_to "#{recipe.name}", recipe, class:"fs-4 link-dark link-underline-light" %>
+  <%= "by: #{recipe.user.id}" %>
```

Comment:
[ ]OPTIONAL: When the usernames save to the table, I recommend you update recipe.user.id to recipe.user.name.